### PR TITLE
React/Setting Up A React Environment: Add additional resources

### PR DESCRIPTION
**Pull request by scottwright-dev**
## Because

This PR adds additional optional resources. The additional video resources offer clear, step-by-step instructions on setting up a Vite React App with ESLint and Prettier in VSCode, and an introduction to React Dev Tools in Chrome. 

## This PR

Adds the following to the additional resources section:
1. A video resource outlining how to correctly set up a Vite React App with EsLint & Prettier in VSCode.
2. A video resource providing a brief overview of React Dev Tools in Chrome and how to get started with it.

## Issue

Closes: N/A

## Additional Information

N/A

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by scottwright-dev** [comment]
Looks like this PR has highlighted some additional formatting issues in the MD file. I am happy to work through these but will take me a bit of time.

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Couple comments below, one being a change request for link text

File: react/introduction/setting_up_a_react_environment.md [hunk 1]
```diff
@@ -133,25 +133,26 @@ We recommend installing this and becoming comfortable with using it as early as
...
+- [What is in the `src` folder?](#delving-deeper)
+- [Why are the React Developer Tools useful?](#developer-tools)
 
 ### Additional resources
 
-This section contains helpful links to related content. It isn’t required, so consider it supplemental.
+This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
-* It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+- [This video](https://www.youtube.com/watch?v=SMbqi1HPprc) is a helpful guide on how to set up Prettier inside your Vite-React projects.
```
[the remark this answers is not in the document]
Comment (scottwright-dev) [reply react/introduction/setting_up_a_react_environment.md:157]: Hi @thatblindgeye, it essentially covers setting up Prettier with a React project properly, so probably more of a "nice to have" more than anything. I just felt it helped contextualise the set up process abit. 

It seems however the PR #27423 could well render that addition less relevant.

[the remark this answers is not in the document]
Comment (thatblindgeye) [reply react/introduction/setting_up_a_react_environment.md:157]: @scottwright-dev If you find it helpful to have an optional visual reference for how to set them up together I won't block it, so I'll leave it up to you on its relevancy/usefulness. If you would like to keep it, let's just update the it to:

```suggestion
- [Vite React App with EsLint & Prettier in VSCode (2023)](https://www.youtube.com/watch?v=SMbqi1HPprc) is a helpful video guide on how to set up Prettier inside your Vite-React projects.
```

**Comment by zachmmeyer** [comment]
@scottwright-dev are you able to resolve your merge conflict?

**Comment by scottwright-dev** [comment]
> @scottwright-dev are you able to resolve your merge conflict?

Hey @zachmmeyer, all done. Appreciate your patience! Best, Scott

**Review by thatblindgeye — APPROVED** [review]

**Pull request merged**